src/Week1/HW01.hs

```diff
@@ -5,35 +5,40 @@ module Week1.HW01 where
 
 -- Get the last digit from a number
 lastDigit :: Integer -> Integer
-lastDigit = undefined 
+lastDigit =  flip mod 10
 
 -- Drop the last digit from a number
 dropLastDigit :: Integer -> Integer
-dropLastDigit = undefined
+dropLastDigit x = (x - lastDigit x) `div` 10
```

Comment:
This does some unnecessary work. You can simplify this (hint: plain integer division should do what you want)